core/lib/config/src/configs/contracts/ecosystem.rs

```diff
@@ -0,0 +1,54 @@
+// External uses
+use serde::{Deserialize, Serialize};
+use zksync_basic_types::Address;
+
+// Unified ecosystem contracts. To be deleted, after contracts config migration
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+pub struct EcosystemContracts {
```

Comment:
These contracts refer to L1 only, why do we need a separate `L1SpecificContracts`?